roles/openshift_setup/tasks/main.yml

```diff
@@ -168,6 +168,31 @@
             additionalTrustedCA:
               name: "registry-cas"
 
+- name: Add insecure registry
+  when: cifmw_update_containers_registry is defined
+  kubernetes.core.k8s:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit)}}"
+    context: "{{ cifmw_openshift_context | default(omit)}}"
+    merge_type: "merge"
+    definition:
+      apiVersion: config.openshift.io/v1
+      kind: Image
+      metadata:
+        name: cluster
+      spec:
+        registrySources:
+          insecureRegistries:
+            - "{{ cifmw_update_containers_registry }}"
+          allowedRegistries:
```

Comment:
it will not raise any issue if they would be repeating? If not, it's fine.
Maybe setting fact  then `new_registry_var | unique`  should be helpful there.

Reply:
Ack, will do.